nautobot/ipam/models.py

```diff
@@ -882,8 +882,9 @@ class IPAddress(PrimaryModel, StatusModel, RoleModelMixin):
         db_index=True,
         help_text="IPv4 or IPv6 host address",
     )
-    broadcast = VarbinaryIPField(null=False, db_index=True, help_text="IPv4 or IPv6 broadcast address")
-    prefix_length = models.IntegerField(null=False, db_index=True, help_text="Length of the Network prefix, in bits.")
+    mask_length = models.IntegerField(
+        null=False, db_index=True, help_text="Length of the Network Configuration Mask, in bits."
+    )
```

Comment:
I'm not sure how I feel about this help text.
```suggestion
        null=False, db_index=True, help_text="Length of the network mask, in bits."
    )
```